libraries/AP_HAL_ChibiOS/Scheduler.cpp

```diff
@@ -662,6 +662,7 @@ uint8_t Scheduler::calculate_thread_priority(priority_base base, int8_t priority
         { PRIORITY_CAN, APM_CAN_PRIORITY},
         { PRIORITY_TIMER, APM_TIMER_PRIORITY},
         { PRIORITY_RCOUT, APM_RCOUT_PRIORITY},
+        { PRIORITY_LED, APM_LED_PRIORITY},
```

Comment:
might be needed in other HALs for completeness

Reply:
Actually RCOUT is also missing so we clearly do not do this currently